backend/src/test/kotlin/org/loculus/backend/controller/submission/SubmissionConvenienceClient.kt

```diff
@@ -287,6 +287,24 @@ class SubmissionConvenienceClient(
         ),
     )
 
+    fun getStatusCount(status: Status, userName: String = DEFAULT_USER_NAME): Int =
+        deserializeJsonResponse<GetSequenceResponse>(
+            client.getSequenceEntries(jwt = generateJwtFor(userName))
+                .andExpect(status().isOk)
+                .andExpect(
+                    content().contentType(MediaType.APPLICATION_JSON_VALUE),
+                ),
+        ).statusCounts[status]!!
+
+    fun getProcessingResultCount(processingResult: ProcessingResult, userName: String = DEFAULT_USER_NAME): Int =
+        deserializeJsonResponse<GetSequenceResponse>(
+            client.getSequenceEntries(jwt = generateJwtFor(userName))
+                .andExpect(status().isOk)
+                .andExpect(
+                    content().contentType(MediaType.APPLICATION_JSON_VALUE),
+                ),
+        ).processingResultCounts[processingResult]!!.toInt()
+
     fun expectStatusCountsOfSequenceEntries(statusCounts: Map<Status, Int>, userName: String = DEFAULT_USER_NAME) {
```

Comment:
IntelliJ tells me that this is not used anymore and can be deleted (this might not actually related to a change of this PR)

Reply:
deleted https://github.com/loculus-project/loculus/pull/3125/commits/f213a237c8b1f190741ff319ee17c4e4d047e2a5